caravel/forms.py

```diff
@@ -653,7 +653,7 @@ def __init__(self, viz):
                 "default": '.3s',
                 "choices": [
                     ('.3s', '".3s" | 12.3k'),
-                    ('.3%', '".3%" | 1234543.210%'),
+                    ('.1%', '".1%" | 1234543.2%'),
```

Comment:
It seems unusual for percentage axis numbers to be shown as 60.000% when 60℅ or 60.0℅ will do.